webrtc.html

```diff
@@ -5946,7 +5948,7 @@ <h2>Methods</h2>
              RTCDegradationPreference                  degradationPreference;
 };</pre>
         <section>
-          <h2>Dictionary <dfn>RTCRtpParameters</dfn> Members</h2>
+          <h2>Dictionary <code><a>RTCRtpParameters</a></code> Members</h2>
```

Comment:
I'm not sure it matters, either in TOC-gen or in styling, but is it really proper to use a h2 inside a h3, and not a h4?

Reply:
h2 is used consistently throughout the spec for these kind of sections.  And yes, it does seem odd.